lib/rubocop/cop/rspec/example_wording.rb

```diff
@@ -106,6 +110,11 @@ def docstring(node)
           )
         end
 
+        def needs_escape?(node)
+          node.value.include?(node.loc.begin.source) ||
```

Comment:
For quoted strings, both begin and end are the same. Do we need to check twice?

Reply:
This is for `%q(...)`, `%Q(...)`, or `%<...>` etc.

```
irb(#<RuboCop::Cop::RSpec::Exampl...):001> node.loc.begin.source
=> "%q("
irb(#<RuboCop::Cop::RSpec::Exampl...):002> node.loc.end.source
=> ")"
```

But on second thought, we must check against `node.loc.begin.source[-1]`, not `node.loc.begin.source`. I'll try to fix that.

Reply:
hmm, since escaping should only be necessary if begin and end are different (e.g. `%!...!! ` may require escaping, while `%(...) ` does not), it seems that only one of them needs to be checked after all.